src/screens/SecretIconsScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FlatList, StyleSheet, Text, View } from 'react-native';
+
+import { RegularText } from '../components';
+import { colors, Icon, normalize } from '../config';
+
+type TIconItem = {
+  iconName: string;
+};
+
+const iconNames = Object.keys(Icon);
+
+const IconItem = ({ iconName }: TIconItem) => {
+  const SelectedIcon = Icon[iconName];
+  return (
+    <View style={styles.iconField}>
+      <Text>
+        <SelectedIcon size={normalize(35)} color={colors.secondary} />
+      </Text>
```

Comment:
Why did you wrap this component with `Text`?

Reply:
In my tests, when I removed the Text component, the icons were displayed without any problems.

Reply:
You're right! I got a error before that seems to be resolved now.